barman/Dockerfile.template

```diff
@@ -6,7 +6,10 @@ RUN apt-get update \
 		rsync \
 		supervisor \
 	&& rm -Rf /var/lib/apt/lists/* \
-	&& sed -i 's/\(.*pam_loginuid.so\)/#\1/' /etc/pam.d/sshd
+	&& sed -i 's/\(.*pam_loginuid.so\)/#\1/' /etc/pam.d/sshd \
+	&& mkdir /home/postgres \
```

Comment:
Good point!  Let's at least use `install -d -m 0700 -o postgres -g postgres /home/postgres` though, instead of the mkdir, chown, chmod combo.